### DISCUSSION TITLE
Disable Hydra Logging by default

### DISCUSSION THREAD

**Discussion by aadimator** [mit-ll-responsible-ai/hydra-zen#640] [Q&A] [answered]
If we use the default `logging` module from python, hydra supports saving the logs in the output directory. But it doesn't work with Loguru. I found a solution where we can save the loguru logs to the runtime output directory created by hydra. 
![image](https://github.com/mit-ll-responsible-ai/hydra-zen/assets/10383542/37072d4f-7558-4219-8246-b67f50d38c5d)

Now I want to disable the default logging file that is created by Hydra, i.e. `_implementations.log`, but I don't know how I can disable this functionality programmatically, instead of through the yaml config files as shown [here](https://hydra.cc/docs/tutorials/basic/running_your_app/logging/). 

Can anyone guide me on how I can disable the default logging by Hydra using Python code? Thanks

**Comment by rsokl** [comment, marked as answer]
You can do

```python
from hydra.conf import HelpConf
hydra_conf = HydraConf(
    job_logging={
        "version": None,
        "root": None,
        "disable_existing_loggers": False,
    },
    hydra_logging={
        "version": None,
        "root": None,
        "disable_existing_loggers": False,
    },
)

```

and use it as demonstrated in [How To: Customize Hydra's Configuration](https://mit-ll-responsible-ai.github.io/hydra-zen/how_to/configure_hydra.html)

I believe that will do the trick.

**Reply by aadimator** [reply]
That works. Thanks a bunch.
